.editorconfig

```diff
@@ -0,0 +1,11 @@
+# About this file, see:
```

Comment:
This is great! Do you know if it works with VSCode? I went through their docs, but they only mention VisualStudio.

Reply:
https://editorconfig.org/#download

https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

<img width="658" alt="Screen Shot 2021-05-06 at 9 02 20 am" src="https://user-images.githubusercontent.com/418747/117220142-fc14cf80-ae49-11eb-8915-8699a008a7c9.png">